lumicks/pylake/force_calibration/tests/test_power_model.py

```diff
@@ -59,7 +58,7 @@ def test_fit_analytic(reference_models, corner_frequency, diffusion_constant, nu
     fit_range = (0, 15000)
     ps = PowerSpectrum(data, sample_rate=len(data))
     ps = ps.in_range(*fit_range)
-    ps = ps.block_averaged(num_blocks=ps.P.size // num_points_per_block)
+    ps = ps.downsampled_by(num_points_per_block)
```

Comment:
I think that you got in a weird git state because this change is in the second commit but the actual API is added in the third commit.

Reply:
Yep. Must've messed something up rebasing. I was actually planning to merge those two commits into one deduplication commit anyhow, so let me do that now. :)